dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/ControllerTest.cs

```diff
@@ -352,6 +365,26 @@ protected void SetSutClaimsPrincipal(User user)
             };
         }
 
+        /// <summary>
+        /// Sets the current assigned Sut with the provided User and Role as the current authenticated user
+        /// </summary>
+        /// <param name="user"></param>
+        /// <param name="userLogin"></param>
+        protected void SetSutClaimsPrincipal(User user, UserLogin userLogin)
+        {
+            if (Sut == null)
+            {
+                throw new InvalidOperationException("[ControllerTest] Sut MUST be set");
+            }
+
+            Sut.ApiClaimsPrincipal = new ApiClaimsPrincipal
+            {
+                RoleId = userLogin.RoleId,
```

Comment:
Should we null check `userLogin` before accessing properties off of it? Based on `MockAuthenticatedUser` which calls this setup method, it's not required

Reply:
Yeah, wouldn't hurt!